core/src/apps/cardano/README.md

```diff
@@ -0,0 +1,205 @@
+# Cardano
+
+MAINTAINER = ??
```

Comment:
> Also, who should be mentioned as the maintainer of the Cardano app? Is that us/me? Or someone from your end? Or both depending on the type of the problem?

This should be the person that modified the code at latest so that is you. It is basically whom to contact if some questions regarding the code occur etc. I guess adding "vacuumlabs" would be fine as well. As you want.